src/main/io/gps.c

```diff
@@ -989,7 +1003,8 @@ static bool UBLOX_parse_gps(void)
         //i2c_dataset.time                = _buffer.posllh.time;
         gpsSol.llh.lon = _buffer.posllh.longitude;
         gpsSol.llh.lat = _buffer.posllh.latitude;
-        gpsSol.llh.alt = _buffer.posllh.altitude_msl / 10 / 100;  //alt in m
+        //gpsSol.llh.alt = _buffer.posllh.altitude_msl / 10 / 100;  //alt in m
```

Comment:
Why is this commented out?

If it's not used any more - remove it, it can always be looked up in the revision history

If it is a temporary change - add a comment explaining this and stating the condition when it should be reinstated.

Reply:
The commented (original) code transferred 1m resolution. I wanted to keep that old knowledge.
But I'll put it in comment.